README.rst

```diff
@@ -2,8 +2,17 @@
                             Tarantool C client libraries
```

Comment:
> As far as Travis-ci is EOL

Travis CI is working (its new infrastructure, travis-ci.com), but has tigher limits than before.

I would say: 'since we're moving from Travis CI to GitHub Actions'.

Tha last nit, excuse me.

Reply:
Resolved.